### ISSUE TITLE
[Suggestion] Separate unbreakable block and normal block into different id

### ISSUE THREAD

**Issue by GoldenTotem** [TeamMetallurgy/Atum2#428]
Unbreakable blocks and normal blocks use the same id but different blockstate, this may cause a problem that you cannot only give block tags to unbreakable blocks.
If you create a new block atum:unbreakable_limestone_brick_large to replace atum:limestone_brick_large with blockstate unbreakable=true, you can give block tags such as minecraft:wither_immune to it to prevent it from being broken. Some mods also use block tags to prevent blocks being broken from mobs. What's more, you can set its hardness to -1 and resistance to 3600000 to make it unbreakable easily.
To sum up, creating new block id using for unbreakable blocks will bring a lot of convenience.

**Comment by GirafiStudios** [comment]
That´s have already been done for newer Minecraft versions (Which is unreleased, as it never was completed).
Can´t be done for 1.16.5, as it would break excisting worlds unfortunately.
